client/push/fcm.js

```diff
@@ -0,0 +1,30 @@
+(function (global, factory) {
```

Comment:
Please follow the coding rule.
There must be no spaces between function name and ().